msgpack-core/src/test/scala/org/msgpack/core/MessageUnpackerTest.scala

```diff
@@ -330,21 +360,28 @@ class MessageUnpackerTest extends MessagePackSpec {
       new SplitTest {val data = testData3(30)}.run
     }
 
-    "read numeric data at buffer boundary" taggedAs("boundary2") in {
+    "read data at buffer boundary" taggedAs("boundary2") in {
```

Comment:
This test case is intended for checking a bug on unpackInteger that found in the past. We should have another test case for string.